RULEngine/Framework.py

```diff
@@ -229,6 +229,8 @@ def _test_vision(self):
     def _kalman_vision(self):
         vision_frames = self.vision.pop_frames()
         new_image_packet = self.image_transformer.update(vision_frames)
+        self.game.field.update_field_dimensions(vision_frames)
+
```

Comment:
doing it every loop seems excessive, even if its every  3 seconds. Do we expect it to change a lot? Maybe find a better way in the future.